src/renderers/shared/shared/__tests__/ReactCompositeComponentState-test.js

```diff
@@ -345,6 +349,15 @@ describe('ReactCompositeComponent-state', () => {
     expect(() => {
       ReactDOM.unmountComponentAtNode(container);
     }).not.toThrow();
+
+    // Stack doesn't warn
+    if (ReactDOMFeatureFlags.useFiber) {
```

Comment:
Why?

Also, if it doesn't warn, do you think this could create warning noise in codebases that already rely on this pattern? It would be nice to check Power Editor and other React-heavy projects for this.